src/frontend/components/TimedTextListItem/index.tsx

```diff
@@ -130,7 +130,7 @@ export const TimedTextListItem = ({ track }: TimedTextListItemProps) => {
         {track.upload_state === uploadState.READY && (
           <React.Fragment>
             &nbsp;/&nbsp;
-            <a href={track.url} download>
+            <a href={track.origin_url}>
```

Comment:
Why remove `download`?

Reply:
I though it was useless. If not we have to add it on other links.

Reply:
In fact this attribute is useless if you use with urls that has not same origin than your domain.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#download